test/e2e/serial/workload_placement_test.go

```diff
@@ -108,23 +109,75 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 			//TODO : filter nodes with SingleNumaNode policy?
 
 			By("Calculate the requirements")
-			requiredRes, err := getWorkloadRequirements(nodeResourceTopologies, suitableNodes)
+			requiredRes, err = getWorkloadRequirements(nodeResourceTopologies, suitableNodes)
 			if err != nil {
 				Skip(fmt.Sprintf("not suitable request found: %v", err))
 			}
 
-			By("Prepare the pod object with the requirements")
-			pod = objects.NewTestPodPause(testFixture.Namespace.Name, "testpod")
-			pod.Spec.SchedulerName = schedulerName
-			pod.Spec.Containers[0].Resources.Limits = requiredRes
-
 		})
 
-		FIt("Dummy test right now", func() {
+		When("creating a guaranteed pod with one container", func() {
+			var guaranteedPod *corev1.Pod
+			var updatedPod *corev1.Pod
+
+			BeforeEach(func() {
+				By("Prepare the pod object with the requirements")
```

Comment:
this cannot fail, hence should probably be `klog.Infof`

Reply:
ok